src/App.spec.js

```diff
@@ -1,26 +1,41 @@
-import { App, DetailComponents } from '../App';
-import configureMockStore from 'redux-mock-store';
+import App from './App';
 import { Provider } from 'react-redux';
-import thunk from 'redux-thunk';
 import 'regenerator-runtime/runtime';
+import { render, screen, waitFor } from '@testing-library/react';
+import * as useUpdateLocationHook from './hooks/useUpdateLocation';
+import store from './app/store';
 import { MemoryRouter, Route, Routes } from 'react-router-dom';
-import { render, screen } from '@testing-library/react';
-import * as useUpdateLocationHook from '../hooks/useUpdateLocation';
-import { defaultQuery } from '../reducers/query/query';
+import { ComplaintDetail } from './components/ComplaintDetail/ComplaintDetail';
 
 jest.mock('highcharts/modules/accessibility');
 jest.mock('highcharts/highmaps');
 
 describe('initial state', () => {
-  test('renders search page', () => {
+  it('renders search page', () => {
+    const updateLocationHookSpy = jest.spyOn(
+      useUpdateLocationHook,
+      'useUpdateLocation',
+    );
+
+    render(
+      <Provider store={store}>
+        <App />
+      </Provider>,
+    );
+
+    expect(updateLocationHookSpy).toBeCalled();
+
+    expect(screen.getByText(/Consumer Complaint Database/)).toBeDefined();
```

Comment:
Not a huge deal, but are we able to use the `toBeIntheDocument()` assertion here instead of `toBeDefined()`? It's a bit more descriptive of what's actually being tested. This would also apply in other tests where `toBeDefined()` is used.

Reply:
You're right. that is more correct.